enterprise/frontend/src/metabase-enterprise/comments/components/Discussion/DiscussionReactions.tsx

```diff
@@ -0,0 +1,63 @@
+import { useCallback } from "react";
+
+import { getCurrentUser } from "metabase/admin/datamodel/selectors";
+import { useSelector } from "metabase/lib/redux";
+import { Flex, Text, Tooltip } from "metabase/ui";
+import type { Comment, CommentReaction } from "metabase-types/api";
+
+import S from "./Discussion.module.css";
+
+type DiscussionReactionsProps = {
+  comment: Comment;
+  onReactionRemove?: (comment: Comment, emoji: string) => void;
+};
+
+export function DiscussionReactions({
```

Comment:
nitpick, but I suggest renaming to `CommentReactions`, since "Discussion" means the whole thread in our case.